src/shared/components/query/QueryStore.ts

```diff
@@ -275,40 +242,18 @@ export class QueryStore
 	@computed get selectableSelectedStudyIds():string[]
 	{
 		let ids:string[] = this._allSelectedStudyIds.keys();
-		const selectableStudies = this.selectableStudiesSet;
+		const selectableStudies = this.selectableStudiesSet.result;

```

Comment:
check to make sure it's not undefined (what is default of this promise?)

Reply:
default is `{}`